src/Installer/redist-installer/targets/GenerateBundledVersions.targets

```diff
@@ -1247,7 +1247,9 @@ Copyright (c) .NET Foundation. All rights reserved.
     <PropertyGroup>
       <MinimumMSBuildVersionFile>$(RedistLayoutPath)sdk/$(Version)/minimumMSBuildVersion</MinimumMSBuildVersionFile>
       <BundledMSBuildPropsFileName>Microsoft.NETCoreSdk.BundledMSBuildInformation.props</BundledMSBuildPropsFileName>
-      <BundledMSBuildVersion>$(MSBuildVersion)</BundledMSBuildVersion>
+      <_ShippingMSBuildVersion>$(MicrosoftBuildVersion)</_ShippingMSBuildVersion>
```

Comment:
This version is from the eng/Versions.props for MSBuild - aka the version of MSBuild that the SDK being built will be shipping.